docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1196 @@
+{
```

Comment:
<p>Is the horizontal axis in days? I find the results here not consistent with the results below. Somewhere I saw you said you didn't know if you were computing the K-M estimator correctly. Maybe you can use <code>KaplanMeierFitter</code> from the <code>lifelines</code> package?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yeah I will give the lifelines package a try. I initially wanted to avoid the external dependency.

Reply:
I like avoiding external dependencies, but it's not too bad for an example

Reply:
My KM estimator was wrong :( I will use the `lifelines` estimator in the docs.